### DISCUSSION TITLE
error with runScenic_2

### DISCUSSION THREAD

**Discussion by GreyRockIQ** [aertslab/SCENIC#289] [R]
Hello
I am running Scenic in R with example Toy dataset and getting this error with runScenic_2_createRegulons

Error in sapply(getDatabases(scenicOptions), dbLoadingAttempt, dbIndexCol = indexCol) : 
object 'indexCol' not found

Any suggestions?

Thanks
GreyRock

**Comment by s-aibar** [comment]
Dear @GreyRockIQ,

Can you confirm which package version are you using of RcisTarget and SCENIC? That parameter was updated in one of the latest versions, so it could be that you have a newer/older version of one of the packages so they are not matching...

e.g.  
`packageVersion("RcisTarget")` , `packageVersion("SCENIC")`

**Reply by GreyRockIQ** [reply]
Hi @s-aibar 
Thanks for your response. Here are the package versions running 

> packageVersion("RcisTarget")
[1] ‘1.14.0’
> packageVersion("SCENIC")
[1] ‘1.3.0’

Please let me know if any changes could help me.
Thanks
GreyRock

**Comment by s-aibar** [comment]
Dear @GreyRockIQ ,

The issue seems to be that you have inconsistent versions: SCENIC 1.3 requires RcisTarget **1.16** (from Bioconductor 3.15, with R 4.2, which were released a few days ago).

If you cannot install the newer versions of R/Bioconductor, then check out the "Setup" vignette, which indicates how to install older versions of SCENIC.

Sorry for the inconvenience!

**Reply by GreyRockIQ** [reply]
Hi @s-aibar 
Thanks for your response. I have installed new versions as above R 4.2, Bioconductor 3.15 and package versions
> packageVersion("RcisTarget")
[1] ‘1.16.0’
> packageVersion("SCENIC")
[1] ‘1.3.0’
 as suggested but I have still ended up at same error

Error in sapply(getDatabases(scenicOptions), dbLoadingAttempt, dbIndexCol = indexCol) : 
object 'indexCol' not found

Could you please let me know if anything else I can change?

Thanks
GreyRock

**Comment by s-aibar** [comment]
Hello,

Sorry! The error was due to a typo in the function in SCENIC!
I think it should now be solved. Please, re-install the package, sorry for the mess!

**Reply by GreyRockIQ** [reply]
Hi @s-aibar 
Thanks for your response.
Re-installing new version passed runSCENIC_2_createRegulons step.
I see a new error at scenicOptions <- runSCENIC_4_aucell_binarize(scenicOptions)

Error in runSCENIC_4_aucell_binarize(scenicOptions) : 
  No cells passed the binarization.

Any suggestions to solve this?
Thanks
GreyRock
